docs/build.md

```diff
@@ -0,0 +1,13 @@
+# How to build and host the Chatterino 2 API
+
+## Build
+
+1. Clone the repo: `git pull https://github.com/Chatterino/api.git`
+2. Move into the directory, fetch the dependencies and build the binary: `cd api && make build`
+3. Start the API: `./api`
+
+## Install systemd unit
+1. Install the pre-packaged systemd unit file: `sudo cp ./docs/example.service /etc/systemd/system/chatterinoapi.service`
```

Comment:
Either apply this change, or rename `./docs/exampleservice` file to something like `./docs/example.service` (or maybe even better, name it already `./docs/chatterinoapi.service`)
```suggestion
1. Install the pre-packaged systemd unit file: `sudo cp ./docs/exampleservice /etc/systemd/system/chatterinoapi.service`
```